#!/usr/bin/env python
PACKAGE = "slaw_arm_navigation"

import roslib;roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()

gen.add("joint_1", double_t, 0, "Joint 1 Value", 0.0100692, 0.0100692, 5.84014)
gen.add("joint_2", double_t, 0, "Joint 2 Value", 0.0100692, 0.0100692, 2.61799)
gen.add("joint_3", double_t, 0, "Joint 3 Value", -0.015708, -5.02655, -0.015708)
gen.add("joint_4", double_t, 0, "Joint 4 Value", 0.0221239, 0.0221239, 3.4292)
gen.add("joint_5", double_t, 0, "Joint 5 Value", 0.110619, 0.110619, 5.64159)

exit(gen.generate(PACKAGE, "ArmConfig", "ArmConfig"))
